Source/Shared/BPUtils.m

```diff
@@ -287,4 +309,19 @@ + (NSString *)trimTrailingParanthesesFromTestName:(NSString *)testName {
     return [testName substringWithRange:regexMatches.firstObject.range];
 }
 
++ (NSDictionary *)loadJsonMappingFile:(NSString *)filePath
+                            withError:(NSError **)errPtr {
+    NSData *data = [NSData dataWithContentsOfFile:filePath
+                                          options:NSDataReadingMappedIfSafe
+                                            error:errPtr];
+    if (!data) {
+        BP_SET_ERROR(errPtr, @"Unable to read the input file");
```

Comment:
This error is not very helpful to the user... what input file couldn't it load? How do I fix the problem?

Reply:
I will add more detail. To quote a wise man I know (re: you), the best error message is that which says what's exactly wrong **and how to fix it**, right?